ros/src/computing/perception/detection/lidar_detector/packages/lidar_point_pillars/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+find_library(NVONNXPARSERS NAMES nvonnxparser)
+if(NVINFER AND NVPARSERS AND NVONNXPARSERS)
+   message("TensorRT is available!")
+   message("NVINFER: ${NVINFER}")
+   message("NVPARSERS: ${NVPARSERS}")
+   message("NVONNXPARSERS: ${NVONNXPARSERS}")
+   set(TRT_AVAIL ON)
+else()
+  message("TensorRT is NOT Available")
+  set(TRT_AVAIL OFF)
+endif()
+
+# set flags for CUDNN availability
+option(CUDNN_AVAIL "CUDNN available" OFF)
+# try to find the CUDNN module
+find_library(CUDNN_LIBRARY
+NAMES libcudnn.so${__cudnn_ver_suffix} libcudnn${__cudnn_ver_suffix}.dylib ${__cudnn_lib_win_name}
+PATHS $ENV{LD_LIBRARY_PATH} ${__libpath_cudart} ${CUDNN_ROOT_DIR} ${PC_CUDNN_LIBRARY_DIRS} ${CMAKE_INSTALL_PREFIX}
+PATH_SUFFIXES lib lib64 bin
+DOC "CUDNN library." )
+if(CUDNN_LIBRARY)
+   message("CUDNN is available!")
+   message("CUDNN_LIBRARY: ${CUDNN_LIBRARY}")
+   set(CUDNN_AVAIL ON)
+else()
+  message("CUDNN is NOT Available")
+  set(CUDNN_AVAIL OFF)
+endif()
+
+if(TRT_AVAIL AND CUDA_AVAIL AND CUDNN_AVAIL)
+
+  find_package(autoware_build_flags REQUIRED)
+  find_package(catkin REQUIRED COMPONENTS
+   roscpp
+   pcl_ros
+   autoware_msgs
+   )
+  set(CMAKE_CXX_STANDARD 11)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+  catkin_package(
+   CATKIN_DEPENDS
+   roscpp
+   pcl_ros
+   autoware_msgs
+   )
+
+  include_directories(
+   include
+   ${catkin_INCLUDE_DIRS}
+  )
+  set(SOURCE_FILES
+   nodes/lidar_point_pillars_node.cpp
+   nodes/point_pillars_ros.cpp
+   )
+
+  add_executable(lidar_point_pillars
+         ${SOURCE_FILES})
+
+  add_dependencies(lidar_point_pillars
+         ${catkin_EXPORTED_TARGETS}
+         )
+
+
+  cuda_add_library(gpu_point_pillars_lib
+         nodes/preprocess_points_cuda.cu
+         nodes/anchor_mask_cuda.cu
+         nodes/scatter_cuda.cu
+         nodes/postprocess_cuda.cu
+         nodes/nms_cuda.cu
+         )
+
+  target_link_libraries(gpu_point_pillars_lib
+   ${CUDA_LIBRARIES}
+  )
+
+  add_library(point_pillars_lib
+         nodes/point_pillars.cpp
+         nodes/preprocess_points.cpp
+         )
+
+   target_link_libraries(point_pillars_lib
+         ${NVINFER}
+         ${NVONNXPARSERS}
+         ${CUDA_LIBRARIES}
+         ${CUDA_CUBLAS_LIBRARIES}
+         ${CUDA_curand_LIBRARY}
+         ${CUDNN_LIBRARY}
+         gpu_point_pillars_lib
+         )
+
+  target_link_libraries(lidar_point_pillars
+         ${catkin_LIBRARIES}
+         point_pillars_lib
+         )
+
+
+  install(TARGETS
+         gpu_point_pillars_lib
+         point_pillars_lib
+         lidar_point_pillars
+         ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+         LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  )
+
+  install(DIRECTORY launch/
+         DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+         PATTERN ".svn" EXCLUDE)
+
+  install(DIRECTORY include/
+         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
```

Comment:
you are probably going to hate me but, can you please move all your includes to `/include/lidar_point_pillars/`

You'll need to change your includes to prefix `lidar_point_pillars/` to all your package includes.

If I'm not clear enough:
Currently you have this
```
├── include
│   ├── anchor_mask_cuda.h
│   ├── common.h
│   ├── nms_cuda.h
│   ├── point_pillars.h
│   ├── point_pillars_ros.h
│   ├── postprocess_cuda.h
│   ├── preprocess_points_cuda.h
│   ├── preprocess_points.h
│   └── scatter_cuda.h
```

Change it to:
```
├── include
│   └── lidar_point_pillars
│            ├── anchor_mask_cuda.h
│            ├── common.h
│            ├── nms_cuda.h
│            ├── point_pillars.h
│            ├── point_pillars_ros.h
│            ├── postprocess_cuda.h
│            ├── preprocess_points_cuda.h
│            ├── preprocess_points.h
│            └── scatter_cuda.h
```
and update your source files, adding the `lidar_point_pillars` prefix:
i.e.
```
--- a/ros/src/computing/perception/detection/lidar_detector/packages/lidar_point_pillars/nodes/anchor_mask_cuda.cu
+++ b/ros/src/computing/perception/detection/lidar_detector/packages/lidar_point_pillars/nodes/anchor_mask_cuda.cu
@@ -16,7 +16,7 @@
 
 //headers in local files
 #include "common.h"
-#include "anchor_mask_cuda.h"
+#include "lidar_point_pillars/anchor_mask_cuda.h"
```

Reply:
just for completeness, please use `git mv` so you don't lose your history.